pdfarranger.spec

```diff
@@ -37,46 +36,23 @@ PDF-Shuffler that aims to "make the project a bit more active".
 %autopatch -p1
 
 %build
-%py3_build
+%py_build
 
 %install
-%{__python3} setup.py install --root %{buildroot}
+%{__python} setup.py install --root %{buildroot}
 
 %find_lang %{name}
 
 %files -f %{name}.lang
-%doc AUTHORS ChangeLog README.md TODO
+%doc README.md
 %license COPYING
 %{_bindir}/%{name}
-%{_datadir}/applications/%{name}.desktop
-%{_datadir}/metainfo/%{name}.appdata.xml
+%{_datadir}/applications/com.github.jeromerobert.pdfarranger.desktop
+%{_datadir}/metainfo/com.github.jeromerobert.pdfarranger.metainfo.xml
 %{_mandir}/man1/%{name}.1.*
 %dir %{_datadir}/%{name}
 %{_datadir}/%{name}/%{name}.ui
```

Comment:
I'm worried that you may end up without menu.ui, does it not warn about the unpackaged files during build?

Additionally you may want explicitly state the new dependencies. From my Fedora spec: 

```
Requires:       python3-dateutil >= 2.4.0
Recommends:     python3-img2pdf >= 0.3.4
```

Cheers and thanks for packaging!

Reply:
Thanks for the info, i fixed it in the next commit.
https://github.com/OpenMandrivaAssociation/pdfarranger/commit/3c0d469d65cd975ebf493c728b58b489d1c24729

Still not finished as I am trying to update all required dependency.

Reply:
We may add ui files in the future therefore I'd recommend to take the whole %{_datadir}/%{name}/ directory or at least %{_datadir}/%{name}/*.ui

For dependencies you can have a look at my Spec for Fedora: https://src.fedoraproject.org/rpms/pdfarranger/blob/master/f/pdfarranger.spec

Just noticed that pypdf2 is not needed any more.

Feel free to ping me with any questions.